src/wallet/rpcwallet.cpp

```diff
@@ -3324,6 +3475,125 @@ UniValue z_getbalance(const UniValue& params, bool fHelp)
+    pools.pushKV("orchard", 99999.99);
+
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("pool_balancess", pools);
+    result.pushKV("minimum_confirmations", /*TODO*/ 9999);
+
+    return result;
+}
+
+UniValue z_getbalanceforaccount(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "z_getbalanceforaccount account ( minconf )\n"
+            "\nReturns the spendable pool balances of the given account."
+            "\nArguments:\n"
+            "1. account      (numeric) The account number.\n"
+            "2. minconf      (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"pool_balances\": {\n"
+            "    \"transparent\": {\n"
+            "        \"valueZat\": amount   (numeric) The amount held in the transparent pool by this account\n"
+            "    \"},\n"
+            "    \"sprout\": {\n"
+            "        \"valueZat\": amount   (numeric) The amount held in the sprout pool by this account\n"
+            "    \"},\n"
+            "    \"sapling\": {\n"
+            "        \"valueZat\": amount   (numeric) The amount held in the sapling pool by this account\n"
+            "    \"},\n"
+            "    \"orchard\": {\n"
+            "        \"valueZat\": amount   (numeric) The amount held in the orchard pool by this account\n"
+            "    \"}\n"
+            "  \"}\n"
+            "}\n"
+            "Result amounts are in units of " + MINOR_CURRENCY_UNIT + ".\n"
+            "Pools for which the balance is zero are not shown.\n"
+            "\nExamples:\n"
+            "\nThe per-pool amount received by account 4 with at least 1 block confirmed\n"
+            + HelpExampleCli("z_getbalanceforaccount", "4") +
+            "\nThe per-pool amount received by account 4 with at least 5 block confirmations\n"
+            + HelpExampleCli("z_getbalanceforaccount", "4 5") +
+            "\nAs a JSON RPC call\n"
+            + HelpExampleRpc("z_getbalanceforaccount", "4 5")
+        );
+
+    if (!fExperimentalOrchardWallet) {
+        throw JSONRPCError(RPC_WALLET_ENCRYPTION_FAILED, "Error: the Orchard wallet experimental extensions are disabled.");
+    }
+    int64_t account = params[0].get_int64();
+    int minconf = 1;
+    if (params.size() > 1) {
+        minconf = params[1].get_int();
+        if (minconf < 0) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Minimum number of confirmations cannot be less than 0");
+        }
+    }
+    UniValue pools(UniValue::VOBJ);
+    pools.pushKV("transparent", 99999.99);
+    pools.pushKV("sprout", 99999.99);
+    pools.pushKV("sapling", 99999.99);
+    pools.pushKV("orchard", 99999.99);
+
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("pool_balances", pools);
```

Comment:
```suggestion
    result.pushKV("pool_balances", pools);
    result.pushKV("minimum_confirmations", minconf);
```
(and also the rename of `pool_balances` if we decide to do that).